app/services/trainees/create_from_hesa.rb

```diff
@@ -222,7 +195,7 @@ def training_initiative
     # course, otherwise use HESA's commencement_date.  This is the start date
     # for trainees who have not transferred courses.
     def itt_start_date
-      hesa_trainee[:itt_commencement_date].presence || hesa_trainee[:commencement_date]
+      hesa_trainee[:itt_start_date]
```

Comment:
I think the above comment can be removed as well, as it refers to `itt_commencement_date`